joomla/joomla-cms
# [4.1] Fix dependency on web application stopping content from booting in CLI mode

Pull Request for Issue #35818 .

### Summary of Changes
Removes unused constructor argument. Luckily also fixing #35818 for now although there's still some thought process to work through there.

### Testing Instructions
Check loading of icons in the frontend of com_content and com_contact continues to work (both create and edit icons)

### Documentation Changes Required
None
